README.md

```diff
@@ -1308,6 +1308,31 @@ in various programming languages.
     by John M. Lucassen  
     ([pdf](https://apps.dtic.mil/sti/pdfs/ADA186930.pdf))
 
+
+## Talks
+
+  ### 2023
+
+  * **Deliminated Continuations, Demystified** (Lambda Days 2023)  
```

Comment:
```suggestion
  * **Delimited Continuations, Demystified** (Lambda Days 2023)  
```

Reply:
:sweat_smile: This is how I find out I've been consistently using the wrong word...